.github/workflows/lint-mixins.yml

```diff
@@ -83,4 +83,8 @@ jobs:
 
       - name: Check for unexpected changes in generated files
         working-directory: ./${{ matrix.mixin }}
-        run: make && git diff --exit-code
+        run: make dashboards_out prometheus_alerts.yaml prometheus_rules.yaml && git diff --exit-code
```

Comment:
Do you want to add a warning or act on the diff in any way here?
Correct me if I'm wrong but this will fire for every mixin that doesn't have generated dashboards present?

Reply:
yes, see #1407.

Reply:
I included all of them in #1407 but consul-mixin due to another reason. I'll add it soon.